invenio_rdm_records/records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -471,14 +471,18 @@
             "type": "object",
             "additionalProperties": false,
             "properties": {
-              "point": {
+              "geometry": {
                 "type": "object",
-                "additionalProperties": false,
                 "properties": {
-                  "lat": {"$ref": "#/definitions/latitude"},
-                  "lon": {"$ref": "#/definitions/longitude"}
+                  "type": {
+                    "type": "string"
+                  },
+                  "coordinates": {
+                    "type": "object"
```

Comment:
Minor: Is this not supposed to be an array?

Reply:
Yeah, forgot to put it here was in the minutes of long ago. I didn't update the jsonschema/tests until we had an agreement. This was cuz I had to jump to another task.